docs/_articles/en/user-guide/org-browser.md

```diff
@@ -7,7 +7,7 @@ The Org Browser displays the available metadata types and their corresponding co
 
 ## Opening the Org Browser
```

Comment:
In the second intro sentence (not specifically changing in this draft), I suggest spelling out Developer Edition org, instead of "dev orgs".